### PR TITLE
Commerce-5739 fix for subscriptions

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentItemLocalServiceImpl.java [hunk 1]
```diff
@@ -51,6 +51,30 @@
 public class CommerceShipmentItemLocalServiceImpl
 	extends CommerceShipmentItemLocalServiceBaseImpl {
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceShipmentItem addCommerceDeliverySubscriptionShipmentItem(
```
Comment (dsitu) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentItemLocalServiceImpl.java:56]: We should name this,
```suggestion
	public CommerceShipmentItem addDeliverySubscriptionCommerceShipmentItem(
```

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java [hunk 2]
```diff
@@ -65,6 +66,55 @@
 public class CommerceShipmentLocalServiceImpl
 	extends CommerceShipmentLocalServiceBaseImpl {
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceShipment addCommerceDeliverySubscriptionShipment(
```
Comment (dsitu) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentLocalServiceImpl.java:71]: We should name this,
```suggestion
	public CommerceShipment addDeliverySubscriptionCommerceShipment(
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Review by riccardo-alberti — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**